transcribed/page0020.txt

```diff
@@ -1,33 +1,33 @@
-001                                                                             PAGE 2
```

Comment:
According to my cut-and-paste method, I have determined that the PAGE on line 1 is at column 80.  If you disagree, please provide some evidence.

Reply:
nope, I believe you. I just didn't notice it or know about the file format at the time.